.github/workflows/nix.yaml

```
@@ -114,25 +111,44 @@
+          # Build the Docker image
+          nix build .#dockerImage --accept-flake-config
+          
+          # Get commit SHA for tag
+          SHORT_SHA=$(git rev-parse --short HEAD)
+          
+          # Set image name with architecture suffix
+          echo "image_name=$(nix eval --raw .#dockerImage.imageName):${SHORT_SHA}-${{ matrix.arch }}" >> $GITHUB_OUTPUT
+
+      - name: Install crane
+        if: steps.check-conditions.outputs.should_build == 'true' && (matrix.system == 'x86_64-linux' || matrix.system == 'aarch64-linux') && needs.check-conditions.outputs.should-push-docker == 'true'
+        run: |
```

actionlint (1.7.4)

shellcheck reported issue in this script: SC2086:info:1:10: Double quote to prevent globbing and word splitting

(shellcheck)